### PR TITLE
LPS-145023 | master

### PR DESCRIPTION
**Pull request by ozysouza**
https://issues.liferay.com/browse/LPS-145023

cc @david-gutierrez-mesa 
cc @Tim-Cao

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/macros/sitenavigation/sitenavigationmenu/NavigationMenusAdmin.macro [hunk 1]
```diff
@@ -112,13 +112,37 @@ definition {
...
+
+				var i = ${i} + 1;
+			}
+
+			SelectFrame.selectFrameTop();
+
+			AssertClick(
+				key_text = "Select",
+				locator1 = "Button#ANY",
+				value1 = "Select");
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/macros/sitenavigation/sitenavigationmenu/NavigationMenusAdmin.macro:145]: Group in addCategoryItem temporarily

File: portal-web/test/functional/com/liferay/portalweb/macros/sitenavigation/sitenavigationmenu/NavigationMenusAdmin.macro [hunk 2]
```diff
@@ -193,6 +217,21 @@ definition {
 		Click(
 			key_itemName = "${itemName}",
 			locator1 = "SiteNavigationMenu#ITEM_ENTRY_CLOSE");
+
+		if (IsTextEqual(locator1 = "Modal#HEADER", value1 = "Delete Item")) {
+			AssertTextEquals.assertPartialText(
+				locator1 = "Modal#BODY",
+				value1 = "The item you want to delete has children that also can be removed.");
+
+			Click.clickAtNotVisible(
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/macros/sitenavigation/sitenavigationmenu/NavigationMenusAdmin.macro:226]: Check.checkNotVisible

File: portal-web/test/functional/com/liferay/portalweb/macros/sitenavigation/sitenavigationmenu/NavigationMenusAdmin.macro [hunk 3]
```diff
@@ -193,6 +217,21 @@ definition {
...
+				key_text = "${actionName}",
+				locator1 = "TextInput#ANY");
+
+			AssertClick(
+				key_text = "Delete",
+				locator1 = "Button#ANY",
+				value1 = "Delete");
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/macros/sitenavigation/sitenavigationmenu/NavigationMenusAdmin.macro:233]:
```suggestion
Button.clickDelete();
```

File: portal-web/test/functional/com/liferay/portalweb/paths/portlet/search/Search.path [hunk 4]
```diff
@@ -126,6 +126,11 @@
 	<td>//div[contains(@class,'container-fluid')]//span[contains(@class,'text-truncate')]/span</td>
 	<td></td>
 </tr>
+<tr>
+	<td>SEARCH_INFO_CATEGORY_TREE</td>
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/paths/portlet/search/Search.path:130]: Refactor SEARCH_INFO to
`//div[contains(@class,'container-fluid')]//span[contains(@class,'text-truncate')]/span | //section[contains(@class,'count-feedback')]//p`

**Review by Tim-Cao — COMMENTED** [review]
Hi @ozysouza, could you update according to comments? Thanks

**Comment by ozysouza** [comment]
> Hi @ozysouza, could you update according to comments? Thanks

Hi @Tim-Cao, PR updated according your comments. Would you have a look at?

**Comment by Tim-Cao** [comment]
@ozysouza, see changes https://github.com/liferay-echo/liferay-portal/compare/c6839b22feca63189530f4bc88c1d857e1d7d3a8...0ef2295e091bcb3a332227b2678d442ba416cad8

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**